lib/rules/no-obscure-array-access.js

```diff
@@ -0,0 +1,23 @@
+import Rule from './_base.js';
+
+export function ERROR_MESSAGE(expression) {
+  return `Obscure expressions are prohibited. ${expression}. Please use Ember's get helper instead. i.e {{get list '0'}} `;
```

Comment:
We don't need to repeat the text in error message. Let's remove that.

Reply:
Fixed.